CMakeLists.txt

```diff
@@ -216,22 +220,17 @@ endif()
 
 # Try to find the third party packages before using our subdirectories
 include(cmake/package_helpers.cmake)
-ginkgo_find_package(GTest "GTest::GTest;GTest::Main" FALSE 1.8.1)
-ginkgo_find_package(gflags gflags FALSE 2.2.2)
-ginkgo_find_package(RapidJSON rapidjson TRUE 1.1.0)
+find_package(GTest 1.10.0)
+find_package(gflags 2.2.2 QUIET)
+find_package(RapidJSON 1.1.0 QUIET)
```

Comment:
GTest has a FindGTest module in CMake, so it will work in any situation and only output "GTest not found" by default. gflags and RapidJSON don't, so if not found, they will output a long error message. We could of course decide to make all of them quiet.

Reply:
Thanks for the explanation. Maybe write a comment about that in that case so that someone doesn't get surprised (or maybe if `FindGTest` somehow gets removed).

Reply:
I added some comments :)